package.json

```diff
@@ -28,7 +28,7 @@
     "typescript": "^2.3.3"
   },
   "jspm": {
-    "main": "src/js-form-validator.ts",
+    "main": "dist/js-form-validator.ts",
```

Comment:
@samiam236 I will admit this isn't necessarily tied to this PR, but snuck it in. It is intentional just to reference the Typescript compilation file instead of the source file.